azurelinuxagent/ga/exthandlers.py

```diff
@@ -501,31 +506,37 @@ def handle_ext_handler(self, ext_handler, etag):
 
             return True
 
+        except ExtensionConfigError as error:
+            # Catch and report Invalid ExtensionConfig errors here to fail fast rather than timing out after 90 min
+            self.__handle_and_report_ext_handler_errors(ext_handler_i, error,
+                                                        report_op=WALAEventOperation.InvalidExtensionConfig,
+                                                        message="[Retryable Error] {0}".format(ustr(error)))
```

Comment:
Generally config errors are retryable because most of the times CRP generates a new config which fixes the issue. 
This is purely based on my experiences with CRP, I unfortunately do not have any data to prove this. 

I mainly added the prefix `[Retryable Error]` here to inform the customer that a retry might fix this issue. But since I dont have any data to prove it, I dont mind removing this

Reply:
Since the time you wrote this comment, did you perhaps get any clarity on this? Is it retryable?

Reply:
Unfortunately no, just to be on the safer side, I'm going to modify the message to say something like "Ran into some error, please retry the operation with modified settings to try to mitigate it" or something similar